src/component/Buttons.js

```diff
@@ -1,32 +1,38 @@
-function Buttons() {
-  return (
-    <div className="calc">
-      <input type="text" placeholder="0" id="answer" />
-      <input type="button" value="AC" className="button" />
-      <input type="button" value="+/-" className="button" />
-      <input type="button" value="%" className="button" />
-      <input type="button" value="÷" className="button orange" />
-
-      <input type="button" value="7" className="button" />
-      <input type="button" value="8" className="button" />
-      <input type="button" value="9" className="button" />
-      <input type="button" value="*" className="button orange" />
-
-      <input type="button" value="4" className="button" />
-      <input type="button" value="5" className="button" />
-      <input type="button" value="6" className="button" />
-      <input type="button" value="-" className="button orange" />
+import React from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
 
-      <input type="button" value="1" className="button" />
-      <input type="button" value="2" className="button" />
-      <input type="button" value="3" className="button" />
-      <input type="button" value="+" className="button orange" />
+const Buttons = ({ clickHandler }) => {
+  const buttonValues = [
+    ['AC', '+/-', '%', '÷'],
+    ['7', '8', '9', '*'],
+    ['4', '5', '6', '-'],
+    ['1', '2', '3', '+'],
+    ['0', '.', '='],
+  ];
 
-      <input type="button" value="0" className="button double" />
-      <input type="button" value="." className="button" />
-      <input type="button" value="=" className="button orange" />
+  return (
+    <div className="buttons">
+      {buttonValues.map((row) => (
+        <div key={uuidv4()} className="button-row">
+          {row.map((button) => (
+            <input
+              key={uuidv4()}
+              value={button}
+              className={`button ${
+                ['+', '-', '*', '÷'].includes(button) && 'orange'
+              }`}
+              onClick={() => clickHandler(button)}
+            />
+          ))}
+        </div>
+      ))}
     </div>
   );
-}
+};
+
+Buttons.propTypes = {
+  clickHandler: PropTypes.func.isRequired,
+};
```

Comment:
- [ ] Kindly make sure that click events are implemented on **all buttons**, currently the buttons when clicked show an error of an unknown function `clickHandler is not a function` , please fix this error so that your calculator component works properly, It would be nice to check your design for where you changed styling because in the previous milestone your calculator component matched the given design please update it👍🏿 
![Screenshot_204](https://github.com/Obote/MathMagician/assets/69638013/bdfeddb8-f64e-4ea1-9513-5869aa7ef773)

- [ ]  Kindly make sure that props are passed correctly from the Calculator.js component to the child component, this will help you practice   what you have learned  in this milestone 👍🏿